canonical-web-and-design/maas-ui
# List, create and update API tokens.

Done:
- Add the actions, selectors, forms etc. to create list and update API tokens. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1500. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1566. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1501.

QA:
- Navigate to your preferences and then API keys.
- Check that you can create, update and delete the keys and that they appear in the list.
